Packs/iDefense/Integrations/FeediDefense/FeediDefense.yml

```diff
@@ -0,0 +1,172 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: iDefense Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- display: API Key
+  hidden: false
+  name: api_token
+  required: true
+  type: 4
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: feedInstanceReputationNotSet
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+    fetched from the feed
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+- defaultvalue: indicatorType
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Which indicator types to fetch
+  display: Indicator Type
+  name: indicator_type
+  options:
+  - IP
+  - Domain
+  - URL
+  required: false
+  type: 16
+- additionalinfo: Severity of the indicator.  Which value to fetch.
+  defaultvalue: '0'
+  display: Indicator Severity
+  hidden: false
+  name: severity
+  options:
+  - '1'
+  - '2'
+  - '3'
+  - '4'
+  - '5'
+  required: false
+  type: 15
+- additionalinfo: '?'
+  display: Threat Type
+  hidden: false
+  name: threat_type
+  options:
+  - Vulnerability
+  - ' Cyber Crime'
+  - ' Cyber Espionage'
+  - ' Hacktivism'
+  required: false
+  type: 16
+- additionalinfo: Confidence about the indicator detailes. Choose from which value
+    of confidence to fetch between 0-100
+  display: Confidence from
+  hidden: false
+  name: confidence_from
+  required: false
+  type: 0
+- additionalinfo: Confidence about the indicator detailes. Choose the highest confidence
+    value to fetch between 0-100
+  display: confidence to
+  hidden: false
+  name: confidence_to
+  required: false
+  type: 0
```

Comment:
didn't we say that this param is redundat?

Reply:
In the api there is an option to give range of values. I can do it just from value without an upper limit, what do you think?

Reply:
I agree. let's rm the upper limit.